common-lib/src/cmr/common/config.clj

```diff
@@ -234,3 +245,6 @@
                (pr-str unknown-vars))
          true)
        false))))
+
+(when (seq (approved-pipeline-documents))
+  (info "Prototype feature toggle detected"))
```

Comment:
Should try to end with a newline. See if your editor has a config to add them automatically. Could we also name the prototype?

Reply:
actually this feature will stay forever, it should say something like "Generic documents support: %s" approved-pipeline-documents

Reply:
thanks yall, i added the newline manually for now, for the editor config situation for VS Code has become complicated.